Chamasma/Chamasma/View/CategoryView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  CategoryView.swift
+//  Chamasma
+//
+//  Created by Mariam Ormotsadze on 22.04.23.
+//
+
+import DesignSystem
+import SwiftUI
+
+public struct CategoryView: View {
+    // MARK: - Properties
+    public let title: String
+    // TODO: pass [CategoryItemModel] when MealCell will be refactored to take model not properties
+    // MARK: - Body
+    public var body: some View {
+        ScrollView {
+            VStack {
+                header
+                categoryItemList
+            }
+        }
+    }
+    private var header: some View {
+        VStack {
+            imageHeader
+            ListSectionHeader(title: title)
+        }
+    }
+    private var imageHeader: some View {
```

Comment:
```suggestion
    }
    
    private var imageHeader: some View {
```

Reply:
Sorry, thought lint did not allow that, I misunderstood error message